subsys/modem/backends/modem_backend_uart_async.c

```diff
@@ -185,8 +185,8 @@ static int modem_backend_uart_async_transmit(void *data, const uint8_t *buf, siz
 	ret = uart_tx(backend->uart, backend->async.transmit_buf, bytes_to_transmit,
 		      CONFIG_MODEM_BACKEND_UART_ASYNC_TRANSMIT_TIMEOUT_MS * 1000L);
 
-	if (ret < 0) {
-		LOG_WRN("Failed to start async transmit");
+	if (ret) {
```

Comment:
why this change?

Reply:
and negative value if it isn't, L188 should be kept as-is, or change it to `ret != 0` if you really want to (explicit comparison is better, and it produces boolean result directly)